lxd/storage/drivers/utils_cgo.go

```diff
@@ -360,25 +372,3 @@ func UnsetAutoclearOnLoopDev(loopFd int) error {
 
 	return nil
 }
-
-func SetDirectIOLoopDev(loopFd int) error {
-	ret, err := C.set_direct_io_loop_device(C.int(loopFd))
-	if ret < 0 {
-		if err != nil {
-			return err
-		}
-		return fmt.Errorf("Failed to set LO_FLAGS_DIRECT_IO")
-	}
-	return nil
-}
-
-func UnsetDirectIOLoopDev(loopFd int) error {
-	ret, err := C.unset_direct_io_loop_device(C.int(loopFd))
-	if ret < 0 {
-		if err != nil {
-			return err
-		}
-		return fmt.Errorf("Failed to unset LO_FLAGS_DIRECT_IO")
-	}
-	return nil
-}
```

Comment:
I'm confused. Why is this deleted and where did it go? If you remove it here why introduce it in the previous commit?

Reply:
I initially included these functions just in case I would need to set the `LO_FLAGS_DIRECT_IO` flag from Go but I ended up not needing them after following @stgraber's approach. I'll remove them from the initial commit!